### DISCUSSION TITLE
How actively maintained is sequelize?

### DISCUSSION THREAD

**Discussion by microsoftly** [sequelize/sequelize#17423] [Help & Questions]
I've noticed that v7 has been in the works for a long time and there have been very few merged feature PRs, it seems like basic meta work is happening.

How maintained is sequelize? Is the team behind it still active or is another solution recommended?

**Comment by WikiRik** [comment]
Hi! Thanks for the question. We've indeed been working on v7 for a long time now. We're almost there for a beta release but at this time the current maintainers have limited time so there is not a lot of progress done over the last few months. For non-production projects, we recommend starting with `@sequelize/core@alpha` so you have the latest features ready
